cqf-fhir-cr/src/test/resources/org/opencds/cqf/fhir/cr/measure/r4/ANC/resources/library/library-ANCBaseConcepts.json

```diff
@@ -0,0 +1,53 @@
+{
+  "resourceType": "Library",
+  "id": "ANCBaseConcepts",
+  "extension": [ {
+    "url": "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/cqfm-softwaresystem",
+    "valueReference": {
+      "reference": "Device/cqf-tooling"
+    }
+  } ],
+  "url": "http://fhir.org/guides/who/anc-cds/Library/ANCBaseConcepts",
+  "name": "ANCBaseConcepts",
+  "title": "ANC Base Concepts",
+  "status": "active",
+  "experimental": false,
+  "type": {
+    "coding": [ {
+      "system": "http://terminology.hl7.org/CodeSystem/library-type",
+      "code": "logic-library"
+    } ]
+  },
+  "description": "Base concepts used throughout the ANC content, including decision support and indicator logic.",
+  "relatedArtifact": [ {
+    "type": "depends-on",
+    "display": "Value set Antenatal care case",
+    "resource": "http://fhir.org/guides/who/anc-cds/ValueSet/antenatal-care-case"
+  }, {
+    "type": "depends-on",
+    "display": "Value set Antenatal care contact",
+    "resource": "http://fhir.org/guides/who/anc-cds/ValueSet/antenatal-care-contact"
+  }, {
+    "type": "depends-on",
+    "display": "Value set Pregnancy condition",
+    "resource": "http://fhir.org/guides/who/anc-cds/ValueSet/pregnancy-condition"
+  }, {
+    "type": "depends-on",
+    "display": "Value set Iron supplement",
+    "resource": "http://fhir.org/guides/who/anc-cds/ValueSet/iron-supplement"
+  }, {
+    "type": "depends-on",
+    "display": "Value set Folic acid supplement",
+    "resource": "http://fhir.org/guides/who/anc-cds/ValueSet/folic-acid-supplement"
+  } ],
+  "content": [ {
```

Comment:
We're trying to reduce bloat in the repo by stripping away unneeded content/metadata for libraries. Check out the example here:
https://github.com/vitorpamplona/clinical-reasoning/blob/master/cqf-fhir-cr/src/test/resources/org/opencds/cqf/fhir/cr/measure/r4/FHIR347/resources/library/FHIR347.json

Notice that the content is now a reference to the CQL content. This means we don't have to do the base64 encode/decode while we're editing.

Reply:
There's a cqf-tooling operation to do that automatically here:
https://github.com/cqframework/cqf-tooling/pull/450

Reply:
I went ahead and did this.